sources/api/apiserver/src/server/mod.rs

```diff
@@ -6,25 +6,29 @@ mod error;
 pub use error::Error;
 
 use crate::datastore::{Committed, FilesystemDataStore, Key, Value};
+use crate::server::controller::dispatch_update_command;
```

Comment:
```suggestion
```
The convention in the "view" here is to use the `controller::` prefix to make the separation as clear as possible.

Reply:
Ah, hehe - I didn't mean in the `use` statement, I mean that when we call `dispatch_update_command()` it could be `controller::dispatch_update_command()` to match convention and make clear the distinction.